.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,134 @@
+        Describe the issue. Explain what you _expected_ to happen and what
+        _actually_ happened.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Reproducing the issue
+      description: |
+        Please provide either **steps to reproduce** or a [**minimal reproducible example**](https://stackoverflow.com/help/minimal-reproducible-example).
+        Providing a minimal reproducible example will help us triage your issue
+        much faster.
+    validations:
+      required: false
+  - type: input
+    attributes:
+      label: Firebase SDK Version
+      description: What version of Firebase has this issue?
+      placeholder: "9.4.1"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Xcode Version
+      description: What version of Xcode is being used?
+      placeholder: "13.3.1"
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Installation Method
+      description: How did you add Firebase to your project?
+      options:
+        - Swift Package Manager
+        - CocoaPods
+        - Zip
+        - Carthage
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Firebase Product(s)
+      description: Which Firebase products are being used?
+      multiple: true
+      options:
+        - AB Testing
+        - Analytics
+        - App Check
+        - App Distribution
+        - Authentication
+        - Crashlytics
+        - Database
+        - DynamicLinks
+        - Firestore
+        - Functions
+        - In App Messaging
+        - Installations
+        - ML Model Downloader
+        - Messaging
+        - Performance
+        - Remote Config
+        - Storage
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Targeted Platforms
+      description: What Apple platform(s) is the issue affecting?
+      multiple: true
+      options:
+        - iOS
+        - tvOS
+        - watchOS
+        - macOS
+        - macCatalyst
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Relevant Log Output
+      description: Please copy and paste any relevant log output.
+      placeholder: |
+        Paste the your logs here. Please redact any personally identifiable
+        information. This will be automatically formatted into code, so no
+        need for backticks.
+      render: shell
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: If using CocoaPods, the project's Podfile.lock
+      description: The `Podfile.lock` can help us debug versioning issues.
+      value: |
+        <!--- Look below for instructions on how to share your Podfile.lock. --->
+
+        <details>
+        <summary>Expand <code>Podfile.lock</code> snippet</summary>
+        <br>
+
+        <!--- Paste the contents of your Podfile.lock *inside* the code block below.
```

Comment:
Posters may not know how to translate "code block" to what's below.

Reply:
Resolved in [2a10e3d](https://github.com/firebase/firebase-ios-sdk/pull/10119/commits/2a10e3d08c4bb272ea14970402d5a804d3237b90).

LMK if you think it needs more work.